### PR TITLE
Update ufs-utils-env

### PR DESCRIPTION
**Pull request by HenryRWinterbottom**
## Description

This PR updates the `ufs_utils/package.py` such that it complies with the stack required of the [UFS_UTILS](https://github.com/ufs-community/UFS_UTILS) applications. It is an extension of [#644](https://github.com/JCSDA/spack-stack/issues/644) and based on discussions with @climbfuji (see https://github.com/JCSDA/spack-stack/pull/645#pullrequestreview-1496061873).

## Issue(s) addressed

This is an extension of `spack-stack` issue [#644](https://github.com/JCSDA/spack-stack/issues/644).

## Dependencies

Spack-stack PR [#644](https://github.com/JCSDA/spack-stack/issues/644)

## Impact

This PR, in conjunction with [#644](https://github.com/JCSDA/spack-stack/issues/644), allows a stack to be compiled to support the `develop` branch of [UFS_UTILS](https://github.com/ufs-community/UFS_UTILS).

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] I have run the unit tests before creating the PR

### CODE REVIEW

**Comment by climbfuji** [comment]
@HenryWinterbottom-NOAA The style checks are failing due to a format change you made. Please revert - thanks.
```
==> Running black checks
--- var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py	2023-08-16 22:41:52.937568 +0000
+++ var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py	2023-08-16 22:42:12.521084 +0000
@@ -13,12 +13,11 @@
 
     homepage = "https://github.com/ufs-community/UFS_UTILS"
     git = "https://github.com/ufs-community/UFS_UTILS.git"
     # There is no URL since there is no code to download.
 
-    maintainers("AlexanderRichert-NOAA", "Hang-Lei-NOAA",
-                "HenryWinterbottom-NOAA")
+    maintainers("AlexanderRichert-NOAA", "Hang-Lei-NOAA", "HenryWinterbottom-NOAA")
 
     version("1.0.0")
 
     depends_on("bacio")
     depends_on("g2")
would reformat var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py
```

**Review by climbfuji — CHANGES_REQUESTED** [review]
Style checks fail.

**Comment by HenryRWinterbottom** [comment]
> @HenryWinterbottom-NOAA The style checks are failing due to a format change you made. Please revert - thanks.
> 
> ```
> ==> Running black checks
> --- var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py	2023-08-16 22:41:52.937568 +0000
> +++ var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py	2023-08-16 22:42:12.521084 +0000
> @@ -13,12 +13,11 @@
>  
>      homepage = "https://github.com/ufs-community/UFS_UTILS"
>      git = "https://github.com/ufs-community/UFS_UTILS.git"
>      # There is no URL since there is no code to download.
>  
> -    maintainers("AlexanderRichert-NOAA", "Hang-Lei-NOAA",
> -                "HenryWinterbottom-NOAA")
> +    maintainers("AlexanderRichert-NOAA", "Hang-Lei-NOAA", "HenryWinterbottom-NOAA")
>  
>      version("1.0.0")
>  
>      depends_on("bacio")
>      depends_on("g2")
> would reformat var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py
> ```
@climbfuji I make the correction via `black`. Please let me know if the issue persists.

**Comment by climbfuji** [comment]
> > @HenryWinterbottom-NOAA The style checks are failing due to a format change you made. Please revert - thanks.
> > ```
> > ==> Running black checks
> > --- var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py	2023-08-16 22:41:52.937568 +0000
> > +++ var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py	2023-08-16 22:42:12.521084 +0000
> > @@ -13,12 +13,11 @@
> >  
> >      homepage = "https://github.com/ufs-community/UFS_UTILS"
> >      git = "https://github.com/ufs-community/UFS_UTILS.git"
> >      # There is no URL since there is no code to download.
> >  
> > -    maintainers("AlexanderRichert-NOAA", "Hang-Lei-NOAA",
> > -                "HenryWinterbottom-NOAA")
> > +    maintainers("AlexanderRichert-NOAA", "Hang-Lei-NOAA", "HenryWinterbottom-NOAA")
> >  
> >      version("1.0.0")
> >  
> >      depends_on("bacio")
> >      depends_on("g2")
> > would reformat var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py
> > ```
> 
> @climbfuji I make the correction via `black`. Please let me know if the issue persists.

Thanks @HenryWinterbottom-NOAA - running the unit tests now. Sorry that this took so long to get to.

**Comment by HenryRWinterbottom** [comment]
> > > @HenryWinterbottom-NOAA The style checks are failing due to a format change you made. Please revert - thanks.
> > > ```
> > > ==> Running black checks
> > > --- var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py	2023-08-16 22:41:52.937568 +0000
> > > +++ var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py	2023-08-16 22:42:12.521084 +0000
> > > @@ -13,12 +13,11 @@
> > >  
> > >      homepage = "https://github.com/ufs-community/UFS_UTILS"
> > >      git = "https://github.com/ufs-community/UFS_UTILS.git"
> > >      # There is no URL since there is no code to download.
> > >  
> > > -    maintainers("AlexanderRichert-NOAA", "Hang-Lei-NOAA",
> > > -                "HenryWinterbottom-NOAA")
> > > +    maintainers("AlexanderRichert-NOAA", "Hang-Lei-NOAA", "HenryWinterbottom-NOAA")
> > >  
> > >      version("1.0.0")
> > >  
> > >      depends_on("bacio")
> > >      depends_on("g2")
> > > would reformat var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py
> > > ```
> > 
> > 
> > @climbfuji I make the correction via `black`. Please let me know if the issue persists.
> 
> Thanks @HenryWinterbottom-NOAA - running the unit tests now. Sorry that this took so long to get to.

@climbfuji No need, at all, to apologize! Thank you for the opportunity to contribute and to learn something new!

File: var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py [hunk 1]
```diff
@@ -15,23 +15,37 @@ class UfsUtilsEnv(BundlePackage):
...
     depends_on("w3emc")
-    depends_on("sfcio")
     depends_on("sigio")
+    depends_on("sfcio")
+    depends_on("zlib")
+    depends_on("hdf5")
+    depends_on("netcdf-c")
+    depends_on("netcdf-fortran")
+    depends_on("netcdf-cxx")
+    depends_on("netcdf-fortran")
```
Comment (ulmononian) [inline var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py]: does netcdf-fortran need to be included twice?

**Review by climbfuji — APPROVED** [review]

**Comment by ulmononian** [comment]
does netcdf-fortran need to be included twice?

File: var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py (hunk 1 above)
Comment (climbfuji) [inline var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py]: Good catch @ulmononian I didn't see that.

**Review by climbfuji — CHANGES_REQUESTED** [review]

**Review by ulmononian — APPROVED** [review]
looks good!!

File: var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py (hunk 1 above)
Comment (climbfuji) [reply var/spack/repos/jcsda-emc-bundles/packages/ufs-utils-env/package.py]: @HenryWinterbottom-NOAA Can you fix the double netcdf-fortran please?

**Review by climbfuji — APPROVED** [review]

**Review by ulmononian — APPROVED** [review]

**Pull request merged**